misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkService.kt

```diff
@@ -5,16 +5,13 @@ import com.google.inject.Key
 import misk.DependentService
 import misk.logging.getLogger
 import org.apache.curator.framework.CuratorFramework
-import javax.inject.Inject
-import javax.inject.Singleton
 
-@Singleton
-internal class ZkService @Inject internal constructor(
+internal class ZkService internal constructor(
   private val curatorFramework: CuratorFramework
 ) : AbstractIdleService(), DependentService {
 
   override val consumedKeys: Set<Key<*>> = setOf()
-  override val producedKeys: Set<Key<*>> = setOf(ZookeeperModule.serviceKey)
+  override val producedKeys: Set<Key<*>> = setOf(ZkLeaseModule.serviceKey)
```

Comment:
won't this end up in multiple instances of ZkService producing the same key? Might be useful to have a test where you start multiple zookeepers

Reply:
fixed this up in a new commit 93d007f

I actually went with reusing the same zookeeper instances in tests. Not sure there is a lot of value in actually having separate instances running